toffy/tiling_utils.py

```diff
@@ -928,19 +929,204 @@ def write_manual_to_auto_map(manual_to_auto_map, save_ann, mapping_path):
     save_ann['annotation'] = save_msg
 
 
-def interactive_remap(manual_to_auto_map, manual_fovs_info,
-                      auto_fovs_info, slide_img, mapping_path,
-                      draw_radius=7, figsize=(7, 7)):
-    """Creates the remapping interactive interface for manually-defined
-    to automatically-generated FOVs
+# TODO: potential type hinting candidate?
+def find_manual_auto_invalid_dist(manual_to_auto_map, manual_auto_dist, dist_threshold=2000):
+    """Finds the manual FOVs that map to auto FOVs greater than `dist_threshold` away
 
     Args:
         manual_to_auto_map (dict):
             defines the mapping of manual to auto FOV names
-        manual_fovs_info (dict):
-            maps each manual FOV to its centroid coordinates
-        auto_fovs_info (dict):
-            maps each automatically-generated FOV to its centroid coordinates
+        manual_auto_dist (pandas.DataFrame):
+            defines the distance between each manual FOV from each auto FOV
+        dist_threshold (float):
+            if the distance (in microns) between a manual-auto FOV pair exceeds this value, it will
```

Comment:
specify units here of distance

Reply:
Not sure I follow, are you talking about something like `in stage coordinate microns`, or is this simply an example that should be used in other docstrings?